spec/models/article_spec.rb

```diff
@@ -5,4 +5,7 @@
     it { should validate_presence_of(:title) }
     it { should validate_presence_of(:content) }
   end
+  describe "associations" do
+    it { should belong_to(:user) }
```

Comment:
**suggestion (non-blocking):** Be more assertive!

How would you feel about using the more assertive `is.expected_to` instead of `should`? You should be more confident that you know what your code will do.

```ruby
    it { is.expected_to belong_to :user }
```

(ditto throughout)

Reply:
When I use, `is.expected_to`, I get this error;

```

Randomized with seed 10184
F

Failures:

  1) Article associations
     Failure/Error: it { is.expected_to belong_to :user }

     NameError:
       undefined local variable or method `is' for #<RSpec::ExampleGroups::Article::Associations "example at ./spec/models/article_spec.rb:10">
     # ./spec/models/article_spec.rb:10:in `block (3 levels) in <top (required)>'

Finished in 0.0126 seconds (files took 1.37 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/models/article_spec.rb:10 # Article associations

```

Reply:
Addressed in PR #47